src/usr/local/www/pkg.php

```diff
@@ -183,9 +183,9 @@ function domTT_title($title_msg) {
 
 // Breadcrumb
 if ($pkg['title'] != "") {
-	if (!$only_edit) {
- 		$pkg['title'] = $pkg['title'] . '/Edit';
-	}
+	/*if (!$only_edit) {						// Is any package still making use of this?? Is this something that is still wanted, considering the breadcrumb policy https://redmine.pfsense.org/issues/5527
```

Comment:
Perhaps I was thinking on pkg_edit.php

I'll merge this and we can look out for any issues then. Thanks.